doc/articles/getting-started/wizard/startup.md

```diff
@@ -0,0 +1,122 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Startup
+---
+
+The main page of the wizard shows and the default template is selected.
+
+![](assets/wizard-intro.jpg)
+
+This is introductory page to the wizard and lets you choose between the blank or default template, and also enables you to customize that selection even further if you click *Customize*.
+
+- **Blank**
+
+    The *Blank* template generates a minimal Uno Platform app that targets the following platforms:
+
+    - Windows
+    - Android
+    - iOS
+    - macOS
+    - GTK
+    - WASM
+
+    Click "Create" or hit <kbd>Enter</kbd> to accept the selected choice and generate the project, or select the Default template.
+
+    This is the structure of the generated blank project:
+
+    ![](assets/blank-project-structure.jpg)    
+
+- **Default**
```

Comment:
I think there's too much detail here and repeats content from the various sections. Perhaps one line for each with a link to the section

Reply:
I was actually doubting about it, because when the user chooses to skip customizing, they actually want to know what's included with the default. And for that they need some context.  
Anyway, I shaved them all off a bit.

Reply:
I can remove the entire thing and just discuss Extensions, and Toolkit briefly and link to the 2nd part of the page.
The reason I included it here is so that the user knows which option is on by default, but I can include this information in the detailed information.
Please let me know.